### DISCUSSION TITLE
Mysql ORDER BY FIELD

### DISCUSSION THREAD

**Discussion by omaresmael** [laravel/framework#48592] [Ideas]
Hey guys

This my sql query sorts the records in the order specified by the values list 

```SQL
order by field (column, value1, value2)
```  

currently, we can achieve this in laravel by using the `orderByRaw()` method and passing the field function to it

```php
->orderByRaw( 'field (column, value1, value2)' )
```

I was thinking of creating a method to achieve the same result with a better API

something like

```php
$column = 'id';
$values = ['value1', 'value2'];
->orderByField($column, $values)
```

let me know if that is a good idea or not, so I can create a PR for it

thank you 
stay awesome

**Comment by parallels999** [comment]
Just add a builder macro
